tools/memap.py

```diff
@@ -449,8 +449,40 @@ def generate_output(self, export_format, file_output=None):
         if file_desc is not sys.stdout:
             file_desc.close()
 
+        # If we've generated JSON formatted output we want to return it to aggregate in the future
+        if json_obj:
+            return json_obj
+
         return True
 
+    def agregare_output(self, outputs, onscreen=True):
```

Comment:
This is just printing mechanism here. And it operates on type specified by `MemapParser`.